.travis.yml

```diff
@@ -4,5 +4,6 @@ node_js:
 sudo: false
 script:
   - npm run lint
+  - cd packages/material-ui-codemod && npm install && cd ../../
```

Comment:
I don't like this idea either. I was looking for the best way to do it. Using the `postinstall` sounds far better. Thanks.